src/oas2/transformers/__tests__/params.test.ts

```diff
@@ -157,7 +157,6 @@ describe('params.translator', () => {
         schema: {
           properties: {
             arr: {
-              description: 'desc',
```

Comment:
Are we sure we want these to be removed?

Reply:
@lottamus Thoughts?

Reply:
I believe these are used around? Prism's not — but I think they should not be removed.